src/parquet/column/properties.h

```diff
@@ -78,10 +78,12 @@ class PARQUET_EXPORT ReaderProperties {
 ReaderProperties PARQUET_EXPORT default_reader_properties();
 
 static int64_t DEFAULT_PAGE_SIZE = 64 * 1024 * 1024;
+static bool DEFAULT_IS_DICTIONARY_ENABLED = true;
```

Comment:
But then we need to have that setting respected in the writers in the same way. I'm fully for the fallback-suggestion you have but I just want to prohibit an inconsistency where the properties already use the fallback way and the writers solely rely on `encoding`.